src/extension.ts

```diff
@@ -51,6 +51,7 @@ export function activate(context: vscode.ExtensionContext) {
           Authorization: `Basic ${auth}`,
         },
       }
+      schemaURL = schemaURL + "/configuration-as-code/schema"
```

Comment:
```suggestion
      const schemaURL = jenkinsURL + "/configuration-as-code/schema"
```

Reply:
Also we should properly trim trailing slash or add trailing slash.
So you can either remove or keep the first `/` in `/configuration-as-code/schema`